src/libs/ReportUtils.ts

```diff
@@ -1286,12 +1286,9 @@ function isClosedExpenseReportWithNoExpenses(report: OnyxEntry<Report>): boolean
 /**
  * Whether the provided report is an archived room
  */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 function isArchivedRoom(report: OnyxInputOrEntry<Report>, reportNameValuePairs?: OnyxInputOrEntry<ReportNameValuePairs>): boolean {
```

Comment:
If we don't need `reportNameValuePairs` anymore, we should remove it as a param as well.
```suggestion
function isArchivedRoom(report: OnyxInputOrEntry<Report>): boolean {
```

If so we should also update it in every place that we used this function.

Reply:
We'll need it in the future because the 'private_isArchived' variable will be stores in `reportNameValuePairs` instead of 'report'. This is just a temporary fix while we're sending the variable through 'report'.

Reply:
Got it. I'll start testing now